1-run-automation-test.md

```diff
@@ -0,0 +1,122 @@
+# Running Automation test with Kobiton
+This guide will demonstrate how to run the automation test on Kobiton devices with TeamCity through GitHub repository.
+## Table of contents
++ [Prerequisites](#prerequisites)
++ [1. Config TeamCity and integrate with GitHub](#1-config-teamcity-and-integrate-with-github)
++ [2. Config automation test script](#2-config-automation-test-script)
++ [3. Run automation test on Kobiton devices](#3-run-automation-test-on-kobiton-devices)
++ [4. Fetch test session data through Kobiton REST API](#4-fetch-test-session-data-through-kobiton-rest-api)
+## Prerequisites 
+- Java(JRE). Supported are:
+  + Oracle java 8 or higher
+  + OpenJDK 8
+- Kobiton account
+## 1. Config TeamCity and integrate with GitHub
+This part will guide you how to setup and configure TeamCity with GitHub. Skip this step if you have already had TeamCity setup.
```

Comment:
Write a sentence to remind users to prepare an empty Github **automation test** repository first.